salt/states/file.py

```diff
@@ -1,16 +1,45 @@
 """
-Operations on regular files, special files, directories, and symlinks
-=====================================================================
 
-Salt States can aggressively manipulate files on a system. There are a number
-of ways in which files can be managed.
+This reference page explains how to use the file module in your state files.
+
+For the functions reference documentation, see :ref:`file-module-functions`.
```

Comment:
The "module functions" or "execution module".

This is the functions reference, because everything is a function.

Reply:
Will change, thanks.